CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing
```

Comment:
I like what you've written. I want to note a couple of the terms surprised me. 
e.g. when I went to read Project-Structure, I was expecting a broader scope than code-structure. 

I wonder what the best place is for talking about the team / organisation structure and building in ways wider than code ?
E.g. are designs / mockups welcome / useful.